tools/runners/run-test-suite-test262.py

```diff
@@ -28,37 +29,86 @@ def get_platform_cmd_prefix():
     return ['python2']  # The official test262.py isn't python3 compatible, but has python shebang.
 
 
-def run_test262_tests(runtime, engine, path_to_test262):
-    if not os.path.isdir(os.path.join(path_to_test262, '.git')):
-        return_code = subprocess.call(['git', 'clone', 'https://github.com/tc39/test262.git',
-                                       '-b', 'es5-tests', path_to_test262])
-        if return_code:
-            print('Cloning test262 repository failed.')
-            return return_code
+def get_arguments():
+    execution_runtime = os.environ.get('RUNTIME', '')
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--runtime', metavar='FILE', default=execution_runtime,
+                        help='Execution runtime (e.g. qemu)')
+    parser.add_argument('--engine', metavar='FILE', required=True,
+                        help='JerryScript binary to run tests with')
+    parser.add_argument('--test-dir', metavar='DIR', required=True,
+                        help='Directory contains test262 test suite')
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument('--es51', action='store_true',
+                       help='Run test262 ES5.1 version')
+    group.add_argument('--es2015', action='store_true',
+                       help='Run test262 ES2015 version')
+
+    return parser.parse_args()
+
+
+def prepare_test262_test_suite(args):
+    if os.path.isdir(os.path.join(args.test_dir, '.git')):
+        return 0
+
+    return_code = subprocess.call(['git', 'clone', '--no-checkout',
+                                   'https://github.com/tc39/test262.git', args.test_dir])
+    if return_code:
+        print('Cloning test262 repository failed.')
+        return return_code
+
+    if args.es2015:
+        git_hash = 'fd44cd73dfbce0b515a2474b7cd505d6176a9eb5'
+    else:
+        git_hash = 'es5-tests'
+
+    proc = subprocess.Popen(['git', 'checkout', git_hash], cwd=args.test_dir)
```

Comment:
Can't we just use `subprocess.call` here?

Reply:
I wanted to use it, but unfortunately only Popen supports cwd argument.

Reply:
`call` supports the same arguments as the Popen.

Reply:
Ah, good to know. I should have read the manual thoroughly.